python/py-notebook/Portfile

```diff
@@ -41,7 +41,8 @@ if {${name} ne ${subport}} {
                         port:py${python.version}-ipykernel \
                         port:py${python.version}-send2trash \
                         port:py${python.version}-terminado \
-                        port:py${python.version}-prometheus_client
+                        port:py${python.version}-prometheus_client \
+                        port:py${python.version}-jupyter_packaging
```

Comment:
adding a dependency to `depends_lib` requires an `revision` increase as well. I assume you verified that it's required at runtime as well (otherwise it should be in `depends_build` which would not require a `revision` change).

Reply:
Hum... After inspecting both jupyter_packaging and notebook, I believe you are right and this is only needed for building. I admit that I realized that it was a required dependency because it failed to build otherwise. I tried to deactivate py311-jupyter_packaging, and jupyter-3.11 notebook is still working fine as far as I can tell. I will therefore move the dependency in `depends_build`.